<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Srinivasans-MacBook-Pro.local" name="com.bigbasket.framework.lab.TestNGLab" tests="1" failures="1" timestamp="9 May 2016 23:27:55 GMT" time="0.813" errors="0">
  <testcase name="signInTest" time="0.813" classname="com.bigbasket.framework.lab.TestNGLab">
    <failure type="org.testng.TestNGException" message="
The data provider is trying to pass 1 parameters but the method com.bigbasket.framework.lab.TestNGLab#signInTest takes 0">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 1 parameters but the method com.bigbasket.framework.lab.TestNGLab#signInTest takes 0
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1337)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- signInTest -->
</testsuite> <!-- com.bigbasket.framework.lab.TestNGLab -->
